geopandas/tools/tests/test_sjoin.py

```diff
@@ -7,11 +7,17 @@
 
 import geopandas
 from geopandas import GeoDataFrame, GeoSeries, read_file, sindex, sjoin
+from geopandas._compat import HAS_RTREE
 
 from pandas.testing import assert_frame_equal
 import pytest
 
 
+@pytest.fixture()
```

Comment:
It is still here :). Can you remove lines 16-20 from this PR? These are not needed. Apart from that it is ready to be merged, CI is green and this new test is passing as it should.

Reply:
Apologies! (forgot to save this first time around...)